datalad_registry/tasks/__init__.py

```diff
@@ -631,3 +636,101 @@ def chk_url_to_update(
         db.session.commit()
 
     return ChkUrlStatus.OK_UPDATED if is_record_updated else ChkUrlStatus.OK_CHK_ONLY
+
+
+class FailedSubmission(TypedDict):
+    """
+    A TypedDict a failed submission of a repo URL to Datalad-Registry
```

Comment:
not a sentence... not sure if worth stating TypedDict type here instead of purpose only

Reply:
Yeah, stating `TypedDict` here is unnecessary. The docstring has been updated.